website/content/docs/connect/cluster-peering/k8s.mdx

```diff
@@ -125,9 +126,9 @@ To peer Kubernetes clusters running Consul, you need to create a peering token a
     $ kubectl apply --filename dialer.yml
     ```
 
-## Deploy and export cluster services
+## Export services between clusters
 
-1. For the service in "cluster-02" that you want to export, add the following [annotations](/docs/k8s/annotations-and-labels#consul-hashicorp-com-connect-service-upstreams) to your service's pods. This service is referred to as "backend-service" in the following steps.
+1. For the service in "cluster-02" that you want to export, add the following [annotations](/docs/k8s/annotations-and-labels#consul-hashicorp-com-connect-service-upstreams) to your service's pods.
 
     <CodeBlockConfig filename="backend-service.yml">
```

Comment:
We should really provide an example YAML file instead of a file with snippets in here and describe which annotations are relevant. Someone reading this doc should follow this end to end without needing to looking at other pages of referring to examples for use with cluster peering.